ocs_ci/deployment/aws.py

```diff
@@ -557,9 +567,13 @@ def run_ansible_playbook(self):
         pem_dst_path = "/openshift-dev.pem"
         pod.upload(rhel_pod_obj.name, pem_src_path, pem_dst_path)
         repo_dst_path = "/etc/yum.repos.d/"
-        repo_file = os.path.basename(constants.OCP4_2_REPO)
+        ocp_version = ".".join(
+            config.DEPLOYMENT.get('installer_version').split('.')[:-2]
+        )
+        repo_file = os.path.basename(constants.OCP_REPO_MAP[ocp_version])
```

Comment:
I thought that it will be here also dynamically and no need to add each version to the constants.
```python
repo = os.path.join(constants.REPO_DIR, f"ocp_{version_with_underscore}.repo")
# where version_with_underscore can be from the mentioned function with separator _
repo_file = os.path.basename(repo)
```

One more thing you can add here is check if file exist or not. If not raise the exception that for this version repo file doesn't exist. So we will directly see that we have to add one.

Anyway I think this same thing has to be done for VmWare and in the consolidation of your and Vijay's PR both should use the same function to do this.  But for VmWare it can be handled in separate PR but when fixing for vmware as well we need to make sure to use the same function and not have this logic on both places.

Reply:
done.